includes/discovery/mempools/fortios.inc.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+if ($device['os'] == 'fortios') {
+    echo 'FORTIOS-MEMORY-POOL: ';
```

Comment:
You need to do a snmp_get here to check the device has mempools available before registering discover_mempool.